src/plugins/event-plugins/TTIPlugin.ts

```diff
@@ -0,0 +1,45 @@
+import { Plugin } from '../Plugin';
+
+import { TTIBoomerang } from '../../time-to-interactive/TTIBoomerang';
+import { TimeToInteractiveEvent } from '../../events/time-to-interactive-event';
+import { TIME_TO_INTERACTIVE_EVENT_TYPE } from '../utils/constant';
+import { PluginContext } from 'plugins/types';
+import { isLongTaskSupported } from '../../utils/common-utils';
+
+export const TTI_EVENT_PLUGIN_ID = 'time-to-interactive';
+
+export class TTIPlugin implements Plugin {
+    getPluginId() {
+        return TTI_EVENT_PLUGIN_ID;
+    }
+    protected context!: PluginContext;
+
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
+    enable(): void {}
+
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
+    disable(): void {}
+
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
+    configure(config: any): void {}
+
+    load(context: PluginContext): void {
+        this.context = context;
+        const ttiBoomerang: TTIBoomerang = new TTIBoomerang();
```

Comment:
suggestion: *"Boomerang"* is another project and doesn't help describe the behavior. Instead, name this class *"TimeToInteractive"*.

Reply:
I believe Boomerang does describe the behavior because it is one of many definitions of TTI (though this class is only a partial implementation of Boomerang)

Reply:
changed naming as behavior has been modified from other project